bangazon-clumsy-penguins/WorkforceManagement
# Ajw computer index

# Name - View All Computers

## Description

IT user will be able to view all Computers when they click on the "Computers" link in the Navbar.

User is able to select details of a computer and is directed to the correct URL (details page is not yet available)

User can select the Computer itself and be directed to the correct URL for details.


## Steps to Test
`git fetch`
`git checkout ajw-computerIndex && git pull`
`start WorkforceMnagement.sln`
Build project.

Navigate to the Computers index via the Computers link on the navbar.
Verify all computers are displayed.
Verify that the details and PC link direct to the proper URL `Computer/Index/{id}` no content will be displayed.

##### Link to Feature Ticket
#15
